decentraland/unity-explorer
# fix: avatar preview exception at auth screen

## What does this PR change?

Fixes #755

## How to test the changes?

1. Launch the explorer
2. Go through auth screen and check everything works as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
